glustercli/cmd/volume.go

```diff
@@ -89,6 +91,8 @@ func init() {
 
 	// Volume Expand
 	volumeExpandCmd.Flags().IntVarP(&flagExpandCmdReplicaCount, "replica", "", 0, "Replica Count")
+	volumeExpandCmd.Flags().IntVarP(&flagExpandCmdDistributeCount, "distribute", "", 0, "Replica Count")
+	volumeExpandCmd.Flags().Int64VarP(&flagExpandCmdSize, "size", "", 0, "Replica Count")
```

Comment:
With `--size <expansion-size>`, is this how much to grow by, the new desired size, or ??? And, is it per brick or whole volume
Also, what are the units?